public/app/config/config.js

```diff
@@ -0,0 +1,29 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
interesting - this is flagged because electron is a dev dependency. wonder if it means we should adjust our eslint settings

Reply:
I think it is better to have the warning (and disable it for electron) in case of other misused libraries.